### PR TITLE
docs: tidy

### PR DESCRIPTION
**Pull request by msanft**
This adds some touch-ups to the docs which I found to be hard to read or understand when first familiarizing with them.

### CODE REVIEW

**Review by m1ghtym0 — APPROVED** [review]

File: docs/docs/getting-started/cluster-setup.md [hunk 1]
```diff
@@ -86,7 +86,10 @@ az group create \
 
 ## Create AKS cluster
 
-First create an AKS cluster:
+First create an AKS cluster. We can't directly create a CoCo-only cluster, so we'll need to create a
```
Comment (katexochen) [inline docs/docs/getting-started/cluster-setup.md:89]: Clusters are never "CoCo-only".

File: docs/docs/getting-started/cluster-setup.md [hunk 2]
```diff
@@ -86,7 +86,10 @@ az group create \
...
+non-CoCo cluster first, and then replace the non-CoCo node pool with a CoCo node pool.
+
+We'll first start by creating the non-CoCo cluster:
```
Comment (katexochen) [inline docs/docs/getting-started/cluster-setup.md:92]: Repetition from 3 lines above.

File: docs/docs/getting-started/cluster-setup.md [hunk 3]
```diff
@@ -115,6 +118,16 @@ az aks nodepool add \
   --workload-runtime KataCcIsolation
 ```
 
+
+Now, we can remove the non-CoCo node pool:
```
Comment (katexochen) [inline docs/docs/getting-started/cluster-setup.md:122]: "Optionally, we can now ..."

File: docs/docs/getting-started/cluster-setup.md (hunk 1 above)
Comment (msanft) [reply docs/docs/getting-started/cluster-setup.md:89]: Would you mind to elaborate?

Comment (katexochen) [reply docs/docs/getting-started/cluster-setup.md:89]: Well, you can always deploy non-confidential workloads on nodes that also enable coco workloads.

**Review by katexochen — APPROVED** [review]
Please squash into two commits: one for language improvements and one for the node pool deletion.

**Pull request merged**